x-pack/plugins/reporting/index.js

```diff
@@ -100,6 +100,11 @@ export const reporting = (kibana) => {
               otherwise: Joi.default(true),
             }),
             chromium: Joi.object({
+              inspect: Joi.boolean().when('$dev', {
```

Comment:
Having this merits testing in non-dev mode. I'm working on getting a build off this branch

Reply:
No build necessary - it just means test with `node scripts/kibana` :D